app/src/main/java/com/example/footprnt/Profile/Adapters/MultiViewAdapter.java

```diff
@@ -172,7 +172,7 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
      * @param position position in adapter the POST item is
      */
     private void configurePostViewHolder(final PostViewHolder vh1, final int position) {
-        final Post post = (Post) items.get(position);
+        Post post = (Post) items.get(position);
```

Comment:
This presents the possibility for an IndexOutOfBoundsException. You should probably check there are enough items in "items" before trying to get it.